### PR TITLE
(Chore) Alternative config setup

### PR DESCRIPTION
**Pull request by janjaap**
This PR contains changes that greatly simplify the configuration service; using a Proxy to be able to throw an error when a configuration property is not available and moving all settings to an external file.

The files `environment.conf.acc.json` and `environment.conf.prod.json` will be removed at a later stage when CI/CD is able to create the `environment.conf.json` file before the Docker image is created.

### CODE REVIEW

File: src/signals/incident/components/IncidentPreview/components/MapSelect/index.js [hunk 1]
```diff
@@ -27,7 +26,7 @@ const MapSelectPreview = ({ value, endpoint, incident }) => (
   <Fragment>
     <Values>{value.join('; ')}</Values>
     <MapSelect
-      geojsonUrl={`${configuration.API_ROOT_MAPSERVER}${endpoint}`}
+      geojsonUrl={endpoint}
```
Comment (boris-arkenaar) [inline src/signals/incident/components/IncidentPreview/components/MapSelect/index.js:29]: Is this intentional, to remove the root?

File: src/signals/incident/components/form/MapSelect/index.js [hunk 2]
```diff
@@ -39,8 +38,7 @@ const MapSelectFormComponent = ({
   };
 
   const latlng = getLatlng(parent.meta);
-  const apiRoot = CONFIGURATION.API_ROOT_MAPSERVER;
-  const url = apiRoot + meta.endpoint;
+  const url = meta.endpoint;
```
Comment (boris-arkenaar) [inline src/signals/incident/components/form/MapSelect/index.js:41]: Same here. This makes a test fail.

File: src/signals/incident/components/IncidentPreview/components/MapSelect/index.js (hunk 1 above)
Comment (janjaap) [reply src/signals/incident/components/IncidentPreview/components/MapSelect/index.js:29]: Yes, the whole endpoint URL comes from the config file

File: src/signals/incident/components/form/MapSelect/index.js (hunk 2 above)
Comment (janjaap) [reply src/signals/incident/components/form/MapSelect/index.js:41]: Ah, I didn't have a look at the tests yet.

**Comment by boris-arkenaar** [comment]
This has been merged into PR #807

**Pull request closed without merging**